cli/cli-meta/src/index.ts

```diff
@@ -43,3 +43,26 @@ export function detectIfCurrentPkgIsExecutable (proc: NodeJS.Process = process):
 export function isExecutedByCorepack (env: NodeJS.ProcessEnv = process.env): boolean {
   return env.COREPACK_ROOT != null
 }
+
+export function getCurrentPackageName (proc: NodeJS.Process = process): string {
```

Comment:
We don't really need all the properties of `NodeJS.Process`. I suggest:

```typescript
interface Proc {
  arch: NodeJS.Architecture
  platform: NodeJS.Platform
  pkg?: unknown
}
```

This would remove the type-cast (`as unknown as NodeJS.Process`) in the unit tests, and therefore aid our future selves in refactoring (it would show us TypeScript errors instead of unit test failures).